cpp/data/installer/ub_tools.sql

```diff
@@ -39,8 +39,9 @@ CREATE TABLE zeder_journals (
 
 -- Table to be used w/ our validate_harvested_records tool:
 CREATE TABLE metadata_presence_tracer (
-    zeder_journal_id INT(11) UNSIGNED,
+    journal_id INT(11) UNSIGNED,
```

Comment:
careful about renaming this column. There are about 55 other potential references to this column in the whole code that should be renamed as well, also you forgot the reference in the CONSTRAINT in this table, and there also needs to be an sql update.

Reply:
... and if i say 55 other potential references, i also mean the rest of ub_tools which is not included in this pull request so far (e.g. ZoteroHarvesterUtil and others)

Reply:
Done

Reply:
i hope you're right